applications/StructuralMechanicsApplication/custom_elements/solid_elements/small_displacement_mixed_strain_displacement_element.cpp

```diff
@@ -0,0 +1,1294 @@
+            rElementalDofList[u_block + 1] = r_node.pGetDof(DISPLACEMENT_Y);
+            rElementalDofList[E_block    ] = r_node.pGetDof(STRAIN_VECTOR_XX);
+            rElementalDofList[E_block + 1] = r_node.pGetDof(STRAIN_VECTOR_YY);
+            rElementalDofList[E_block + 2] = r_node.pGetDof(STRAIN_VECTOR_XY);
+        }
+    } else if (dim == 3) {
+        for(IndexType i_node = 0; i_node < n_nodes; ++i_node) {
+            const auto &r_node = r_geometry[i_node];
+            const SizeType u_block = i_node * dim;
+            const SizeType E_block = i_node * strain_size + u_size;
+            rElementalDofList[u_block    ] = r_node.pGetDof(DISPLACEMENT_X);
+            rElementalDofList[u_block + 1] = r_node.pGetDof(DISPLACEMENT_Y);
+            rElementalDofList[u_block + 2] = r_node.pGetDof(DISPLACEMENT_Z);
+            rElementalDofList[E_block    ] = r_node.pGetDof(STRAIN_VECTOR_XX);
+            rElementalDofList[E_block + 1] = r_node.pGetDof(STRAIN_VECTOR_YY);
+            rElementalDofList[E_block + 2] = r_node.pGetDof(STRAIN_VECTOR_ZZ);
+            rElementalDofList[E_block + 3] = r_node.pGetDof(STRAIN_VECTOR_XY);
+            rElementalDofList[E_block + 4] = r_node.pGetDof(STRAIN_VECTOR_YZ);
+            rElementalDofList[E_block + 5] = r_node.pGetDof(STRAIN_VECTOR_XZ);
+        }
+    }
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void SmallDisplacementMixedStrainDisplacementElement::Initialize(
+    const ProcessInfo &rCurrentProcessInfo)
+{
+    KRATOS_TRY
+
+    // Initialization should not be done again in a restart!
+    if (!rCurrentProcessInfo[IS_RESTARTED]) {
+        // Integration method initialization
+
+
+        if (GetProperties().Has(INTEGRATION_ORDER) ) {
+            const SizeType integration_order = GetProperties()[INTEGRATION_ORDER];
+            switch (integration_order)
+            {
+            case 1:
+                mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_GAUSS_1;
+                break;
+            case 2:
+                mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_GAUSS_2;
+                break;
+            case 3:
+                mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_GAUSS_3;
+                break;
+            case 4:
+                mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_GAUSS_4;
+                break;
+            case 5:
+                mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_GAUSS_5;
+                break;
+            default:
+                KRATOS_WARNING("SmallDisplacementMixedStrainDisplacementElement") << "Integration order " << integration_order << " is not available, using GI_LOBATTO_1" << std::endl;
+                mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_LOBATTO_1;
```

Comment:
I think it is possible to directly specify the quadrature. I think that it is more convenient in this case to do so rather than the order.

Reply:
The quadrature enum is exposed to python in here (we forgot to add the Lobatto one by the way)

https://github.com/KratosMultiphysics/Kratos/blob/c134659e95cd084736cf33f9b40b209c3366f256/kratos/python/add_geometry_data_to_python.cpp#L28-L38

I think we can hack the `INTEGRATION_ORDER` variable to take the value of the enum.